ReactWindows/ReactNative.Shared/UIManager/BaseViewManager.cs

```diff
@@ -16,7 +24,8 @@ namespace ReactNative.UIManager
     /// </summary>
     /// <typeparam name="TFrameworkElement">Type of framework element.</typeparam>
     /// <typeparam name="TLayoutShadowNode">Type of shadow node.</typeparam>
-    public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> : ViewManager<TFrameworkElement, TLayoutShadowNode>
+    public abstract class BaseViewManager<TFrameworkElement, TLayoutShadowNode> :
```

Comment:
Maybe not all the view managers, but at least this BaseViewManager.  This is the fundamental building block for almost all ViewManagers, so maybe it makes sense for it to have it's own clean base API for each platform.

---
In reply to: [84543581](https://github.com/ReactWindows/react-native-windows/pull/804#discussion_r84543581) [](ancestors = 84543581)